exercises/practice/two-bucket/two_bucket.py

```diff
@@ -1,2 +1,55 @@
+# def measure(bucket_one, bucket_two, goal, start_bucket):
```

Comment:
Shouldn't this file be stubs and example.py be this solution?

Reply:
GAH!  **good catch!**.  Pushed fix.  Thank you so much for being my safety net!!